File: 0231-power-of-two/0231-power-of-two.c
bool isPowerOfTwo(long long int n) 
{
    if (n%2==1 && n!=1) return false;
    else
    {
        for(int i=0;i<=sqrt(n);i++)
        {
            if ((int)pow(2,i)==n || n==8) return true;
            if ((int)pow(2,i)>n) break;
        }
        return false;
    }
}

File: 0231-power-of-two/0231-power-of-two.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        import math
        if (n%2==1 and n!=1): return False
        else:
            for i in range(0,int(math.sqrt(abs(n))+1)):
                if (int(pow(2,i))==n or n==8): return True
                if (int(pow(2,i))>n): break
            return False